### ISSUE TITLE
Missing Required JogAmp Repo

### ISSUE THREAD

**Issue by BenjaminHolland** [micycle1/processing-core-4#12]
Current build requires org.jogamp.gluegen:gluegen-rt-main:2.5.0-rc-20230523, which (apparently) isn't available from maven central. Adding the jogamp maven fixes it, (kinda). 

```
 maven {
  url=uri("https://jogamp.org/deployment/maven/")
  }
  ```

**Comment by BenjaminHolland** [comment]
Is this addressable upstream?

**Comment by BenjaminHolland** [comment]
```
 Could not find org.jogamp.gluegen:gluegen-rt-main:2.5.0-rc-20230523.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/org/jogamp/gluegen/gluegen-rt-main/2.5.0-rc-20230523/gluegen-rt-main-2.5.0-rc-20230523.pom
       - https://jitpack.io/org/jogamp/gluegen/gluegen-rt-main/2.5.0-rc-20230523/gluegen-rt-main-2.5.0-rc-20230523.pom
     Required by:
         project : > com.github.micycle1:processing-core-4:4.3
```

**Comment by micycle1** [comment]
The repo's pom (Maven) already includes the jogamp repository:

```
<repositories>
	<repository>
		<name>jogamp</name>
		<id>jogamp</id>
		<url>https://jogamp.org/deployment/maven/</url>
	</repository>
</repositories>
```


Are you using Gradle? I wonder if Jitpack doesn't handle the Maven repository definitions when used as a Gradle artefact.

**Comment by micycle1** [comment]
Actually, the 4.3 artefact had jogl at `2.5.0-rc-20230523` as you stated; `2.5.0` was in main branch only.

I've re-released 4.3 as 4.3.1. The build now includes JOGL at v2.5.

**Comment by BenjaminHolland** [comment]
> The repo's pom (Maven) already includes the jogamp repository:
> 
> ```
> <repositories>
> 	<repository>
> 		<name>jogamp</name>
> 		<id>jogamp</id>
> 		<url>https://jogamp.org/deployment/maven/</url>
> 	</repository>
> </repositories>
> ```
> 
> Are you using Gradle? I wonder if Jitpack doesn't handle the Maven repository definitions when used as a Gradle artefact.

Yes, using gradle. There are some other issues with this as well, including the underlying dependencies for jogl not being resolved. Might make a PR with gradle instructions once I get it to work.

**Comment by octobeard** [comment]
I also ran into this today - the fix was adding the additional repo source @BenjaminHolland suggested in the original post. Found in version 4.3.3, the latest at the time of this post.
